babel.config.json

```diff
@@ -14,5 +14,6 @@
         "labelFormat": "[filename]--[local]"
       }
     ]
-  ]
+  ],
+  "plugins": ["babel-plugin-named-exports-order"]
```

Comment:
Can we put this only in storybook config ?

Reply:
done